src/log.rs

```diff
@@ -0,0 +1,21 @@
+use std::io::Write;
+
+fn log<W: Write>(writer: &mut W, message: &str) {
```

Comment:
This function exists, and is leveraged from the `log_conditionally` function, as we'll need to add some other loggers down the road that will also leverage this function (i.e. to follow DRY).